tests/overturemapsdownloader-wheel/pyproject.toml

```diff
@@ -0,0 +1,13 @@
+[tool.poetry]
+name = "overturemapsdownloader-wheel"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+
+[tool.poetry.dependencies]
+python = ">=3.10,<4"
+overturemapsdownloader = "^0.1.9"
```

Comment:
This seems overkill versus a test that just depends on `dask = { ..., extras = ["dataframe"] }`.

I'll see if I can't fix that up and we can get this merged.

Reply:
Fixed.